src/content/doc-surrealql/clauses/omit.mdx

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_position: 1
+sidebar_label: OMIT
+title: OMIT clause | SurrealQL
+description: The `OMIT` clause is used to omit fields from the result set 
+---
+
+# `OMIT` clause
+
+The `OMIT` clause is used to omit fields from the result set which can be particularly useful when querying large datasets.
```

Comment:
How about "when querying the majority of, but not all of the fields of a table"?